### ISSUE TITLE
python autocomplete insert whole word instead of just the missing bit (qmd)

### ISSUE THREAD

**Issue by kompre** [quarto-dev/quarto-vscode#111]
Hi, 

when I hit tab to autocomplete something inside a python code chunk, instead of just the completion, I get the whole word resulting in a wrong command.

before:
![image](https://user-images.githubusercontent.com/11796345/199614877-9ca4a911-ecf7-44ac-8f2a-5116a821c32d.png)

after:
![image](https://user-images.githubusercontent.com/11796345/199614962-82afc775-9ac8-4ee2-9afc-66f18db9275c.png)

This happens both on a windows machine and a linux one. It doesn't happen on .ipynb file

`Quarto extension: v1.54.0`

**on linux:**
```
Version: 1.72.2
Commit: d045a5eda657f4d7b676dedbfa7aab8207f8a075
Date: 2022-10-12T22:16:26.920Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 5.19.0-76051900-generic
Sandboxed: No
```

**on windows:**
```
Version: 1.72.2 (system setup)
Commit: d045a5eda657f4d7b676dedbfa7aab8207f8a075
Date: 2022-10-12T22:15:18.074Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No
```

**Comment by kompre** [comment]
Sooo... no love for this issue? Really doesn't happen to anyone else?

**Comment by jjallaire** [comment]
@kompre Thanks again for reporting, this should be resolved here (https://github.com/quarto-dev/quarto-vscode/commit/1755d70ca197bdd68413be9b6a81c8133bbd21b3) and available in the new v1.56.0 release.
